tests/test_db.py

```diff
@@ -120,6 +120,28 @@ def test_exists_db(tmp_path):
     assert db.exists(db_path)
 
 
+def test_last_modified(tmp_path):
+    url_zst = "https://community-tc.services.mozilla.com/api/index/v1/task/project.bugbug.data_commits.latest/artifacts/public/prova.json.zst"
+    url_version = "https://community-tc.services.mozilla.com/api/index/v1/task/project.bugbug.data_commits.latest/artifacts/public/prova.json.version"
+
+    db_path = tmp_path / "prova.json"
+    db.register(db_path, url_zst, 42)
+
+    responses.add(responses.GET, url_version, status=200, body="42")
+
+    responses.add(
+        responses.HEAD,
+        url_zst,
+        status=200,
+        headers={},
+    )
+
+    # when current db and remote db versions are same, Last-Modified header is not present at the remote db.
+    # then last_modified function should raise Exception.
+    with pytest.raises(Exception, match="Last-Modified is not available"):
+        db.last_modified(db_path)
```

Comment:
It should *not* raise an exception when the versions are the same

Reply:
If versions are same, utils.get_last_modified function is called.
But in utils.get_last_modified function if "Last-Modified" is not present in headers tuple/list then it is returning None.

And last_modified function raises Exception if None.

So, we need to remove that Exception if None.

Reply:
Should we remove/keep Exception if header not found

Reply:
I think we should raise different exception for it.

Reply:
Ah ok, it makes sense then, let's keep this test but please rename it so it's clearer (`test_last_modified_header_not_present`).

Reply:
Actually, it is worth having this test, but it should be added in a separate PR because it's not directly related to version change.
Can you remove it from this PR and add it in a separate PR?

Reply:
Ok